build.gradle

```diff
@@ -69,68 +69,45 @@ import org.gradle.internal.os.OperatingSystem;
 
 tasks.withType(Test) {
     outputs.upToDateWhen { false } // tests will always rerun
-    useTestNG()
+
+    // Always run serially because there are some very badly behaved tests in HTSJDK that
+    // will cause errors and even deadlocks if run multi-threaded
+    maxParallelForks = 1
 
     // set heap size for the test JVM(s)
     minHeapSize = "1G"
     maxHeapSize = "2G"
 
     jvmArgs '-Djava.awt.headless=true'  //this prevents awt from displaying a java icon while the tests are running
 
-    if (System.env.CI == "true") {  //if running under a CI output less into the logs
```

Comment:
It's only critical if it starts killing travis runs again due to the verbosity.  If it's a horrible pain to figure out (which I certainly imagine it could be... I'm not even sure what's generating test output now, is it scalatest, testNG being invoked by scalatest?) then I'd say it's fine as is.  Thanks for looking into it.  If it really becomes problematic on travis we can pipe the test output into `dev/null` or `grep fail` or something like that.